cpg_workflows/stages/alignment/align.py

```diff
@@ -3,9 +3,12 @@
 """
 
 import logging
+from enum import Enum
+from logging import config
+from pickle import MARK
```

Comment:
lots of unused imports here, not sure how this passed linting checks
![Uploading Screenshot 2024-07-04 at 9.23.52 AM.png…]()